.github/workflows/coccicheck.yaml

```diff
@@ -1,7 +1,10 @@
 name: BPF checks
 
 on:
-  - pull_request
+  pull_request: []
+  push:
+    branches:
```

Comment:
nit: this can be just one line `branches: [ master ]`, if you like :)

Reply:
It can be `push: { branches: [ master ] }`, but I find that less readable...

Reply:
I'm expecting we'll soon want to add the backport branches there, so probably not worth fixing.

Reply:
sure!